internal/db/schema/migrations/oss/postgres/44/05_key_rewrap.up.sql

```diff
@@ -0,0 +1,23 @@
+-- make the required schema changes to upgrade the dependency:
```

Comment:
Given that there are no other migrations in `44`, the filename should be `01_key_rewrap.up.sql`.

Reply:
fixed